spec/features/unauthenticated_user/unauthenticated_user_can_visit_ind_item_page_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.